DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/ContractDetails/ContractDetailsViewModel.cs

```diff
@@ -0,0 +1,115 @@
+            TrainersColour = GetColourFromPercentageFilled(trainers, centreDetails.TrainerSpots);
+
+            CmsAdministrators = DisplayStringHelper.GenerateNumberWithLimitDisplayString(
+                cmsAdministrators,
+                centreDetails.CmsAdministratorSpots
+            );
+            CmsAdministratorsColour = GetColourFromPercentageFilled(
+                cmsAdministrators,
+                centreDetails.CmsAdministratorSpots
+            );
+
+            CmsManagers = DisplayStringHelper.GenerateNumberWithLimitDisplayString(
+                cmsManagers,
+                centreDetails.CmsManagerSpots
+            );
+            CmsManagersColour = GetColourFromPercentageFilled(
+                cmsManagers,
+                centreDetails.CmsManagerSpots
+            );
+
+            ContentCreators = DisplayStringHelper.GenerateNumberWithLimitDisplayString(
+                contentCreators,
+                centreDetails.CcLicenceSpots
+            );
+            ContentCreatorsColour = GetColourFromPercentageFilled(
+                contentCreators,
+                centreDetails.CcLicenceSpots
+            );
+
+            CustomCourses = DisplayStringHelper.GenerateNumberWithLimitDisplayString(
+                numberOfCourses,
+                centreDetails.CustomCourses
+            );
+            CustomCoursesColour = GetColourFromPercentageFilled(numberOfCourses, centreDetails.CustomCourses);
+
+            ServerSpace = DisplayStringHelper.GenerateBytesLimitDisplayString(
+                centreDetails.ServerSpaceUsed,
+                centreDetails.ServerSpaceBytes
+            );
+            ServerSpaceColour = GetColourFromPercentageFilled(
+                centreDetails.ServerSpaceUsed,
+                centreDetails.ServerSpaceBytes
+            );
+        }
+
+        public string Administrators { get; set; }
+
+        public string CmsAdministrators { get; set; }
+        public string CmsAdministratorsColour { get; set; }
+
+        public string CmsManagers { get; set; }
+        public string CmsManagersColour { get; set; }
+
+        public string ContentCreators { get; set; }
+        public string ContentCreatorsColour { get; set; }
+
+        public string Trainers { get; set; }
+        public string TrainersColour { get; set; }
+
+        public string Supervisors { get; set; }
+
+        public string CustomCourses { get; set; }
+        public string CustomCoursesColour { get; set; }
+
+        public string ServerSpace { get; set; }
+        public string ServerSpaceColour { get; set; }
+
+        private string GetColourFromPercentageFilled(long number, long limit)
+        {
+            if (limit == 0 && number == 0)
+            {
+                return "grey";
+            }
+
+            var usage = (double)number / limit;
+            if (0 <= usage && usage < 0.6)

```

Comment:
Could do with a space between the start of this if statement and the line above it.